dotCMS/src/integration-test/java/com/dotcms/rest/api/v1/versionable/VersionableResourceTest.java

```diff
@@ -202,4 +208,167 @@ public void test_deleteVersion_limitedUserWithoutEditPermission_contentlet_retur
         resource.deleteVersion(getHttpRequest(limitedUser.getEmailAddress(),"admin"),response,
                 contentlet.getInode());
     }
+
+    /**
+     * Method to test: findVersion in the VersionableResource
+     * Given Scenario: Create and edit a container, this will create an old version of,
+     *                  using the endpoint find the version (the old one).
+     * ExpectedResult: The endpoint should return 200. Check the versionableView against the container.
+     *
+     */
+    @Test
+    public void test_findVersion_container_success() throws DotSecurityException, DotDataException {
```

Comment:
I see `template, contentlet, container` in the tests but I see also `links` as one of the VersionableFinderStrategies. Shouldn't it be added to the tests also?